src/lib/logs/logs.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import gql from 'graphql-tag';
+
+/**
+ * Internal dependencies
+ */
+import API from 'lib/api';
+
+const QUERY_ENVIRONMENT_LOGS = gql`
+	query App( $appId: Int, $envId: Int ) {
+		app( id: $appId ) {
+			environments( id: $envId ) {
+				id
+				logs( type: String, limit: Int ) {
```

Comment:
For it to work, I had to change this to:

```
logs( type: $type, limit: $limit ) {
```

And also the query definition:

```
query App( $appId: Int, $envId: Int, $type: String, $limit: Int ) {
```

(By the way, maybe we could call the query "GetAppLogs", or something like that, so if it gets logged anywhere it's more informative)

Reply:
Thank you for looking into that! I was using the current Parker implementation for testing, which doesn't have the filters yet. I will fix that!